ktor-server/ktor-server-core/jvm/src/io/ktor/routing/RouteSelector.kt

```diff
@@ -92,6 +92,12 @@ public data class RouteSelectorEvaluation(
         public val Constant: RouteSelectorEvaluation =
             RouteSelectorEvaluation(true, RouteSelectorEvaluation.qualityConstant)
 
+        /**
+         * Route evaluation succeeded for a transparent value
```

Comment:
This needs a clarification what exactly "transparent" means, an example or something